### ISSUE TITLE
PANTHER:PTN000074809 (PTHR10689)

### ISSUE THREAD

**Issue by sjm41** [geneontology/go-annotation#2829]
PANTHER:PTN000074809 adds "prostaglandin-E synthase activity" and "prostaglandin biosynthetic process" to all genes under PTHR10689

This is based on IDA/IMP annotations to these 3 genes within PTHR10689:
Human PTGES (O14684)
Rat Ptges (Q9JHF3)
Pig PTGES (Q2TJZ7)

But PTHR10689 contains more than just PTGES orthologs. For example:
- for human, it also includes MGST1 (Microsomal glutathione S-transferase 1) [P10620], which has IDA GO annotations for "glutathione peroxidase activity" & "glutathione transferase activity" (but no prostaglandin-related terms)
- ans similarly for rat, it also includes Mgst1 [P08011], which also has an IDA annotation to "glutathione transferase activity" (and no prostaglandin-related terms)

Bottom line - the 2 prostaglandin-related GO terms should not be propagated to all members of PTHR10689. Not sure if you want to split the PTHR10689 group, or just remove the GO terms from the PAINT node??

**Comment by pgaudet** [comment]
@sjm41 I dont know how you see that - for eg O14684 is not annotated to "prostaglandin-E synthase activity" and "prostaglandin biosynthetic process"  
https://www.ebi.ac.uk/QuickGO/annotations?geneProductId=O14684&evidenceCode=ECO:0000318&evidenceCodeUsage=descendants

??? 

(I'll still have a look at the family - but I was curious to know what you are looking at)

**Comment by sjm41** [comment]
HI Pascale,

The URL you pasted is filtered for IBA terms.
If you use the following URL, you'll see the prostaglandin-related IDA terms (for the human protein):
https://www.ebi.ac.uk/QuickGO/annotations?geneProductId=O14684&aspect=molecular_function,biological_process

Steven

**Comment by sjm41** [comment]
Ah sorry, maybe I misunderstood your point - I think you're saying that the prostaglandin-related terms are NOT being annotated back to human PTGES (O14684) via the IBA/PAINT pipeline.
Yes, I see that now - for all three of the proteins that are the source of the annotations (O14684, Q9JHF3, Q2TJZ7).

And looking again at the MGST1 orthologs (P10620, P08011], I now see that their prostaglandin-related IBA GO annotations have a NOT qualifier via PTN000074809 - sorry, I must have missed this before.

So, given the above, my question becomes:
why are the three fly genes in the PTHR10689 group getting annotated with positive annotations to the two prostaglandin-related terms:
- Mgstl (Q8SY19) (https://www.ebi.ac.uk/QuickGO/annotations?geneProductId=Q8SY19&aspect=molecular_function,biological_process)
- CG33177 (Q86B54) (https://www.ebi.ac.uk/QuickGO/annotations?geneProductId=Q86B54&aspect=molecular_function,biological_process)
- CG33178 (X2JF29) (https://www.ebi.ac.uk/QuickGO/annotations?geneProductId=X2JF29&aspect=molecular_function,biological_process)

**Comment by sjm41** [comment]
tagging @hattrill

**Comment by pgaudet** [comment]
@sjm41 There is something odd about this tree. It looks different in PAINT (we have NOT/IKR annotations). 

I have asked @thomaspd and @huaiyumi to have a look.

Thanks, Pascale

**Comment by pgaudet** [comment]
I fixed the annotations. Not sure why this family was curated like this.

**Comment by sjm41** [comment]
Thanks Pascale - yes, I was also confused by the curation, but glad it's fixed now!
